.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: pytorch-widedeep
+
+on:
+  push:
+#    branches: [ $default-branch ]
+  pull_request:
+#    branches: [ $default-branch ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.7.9, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install black flake8 pytest-cov codecov .
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Code Style (Black/Flake8)
```

Comment:
The code style test is not python version dependent, it should be a separate job I think. When using Travis we only run it once

Reply:
I think you are right, I created a separate job for it